xilem/src/style.rs

```diff
@@ -32,6 +33,24 @@ pub trait Style: Sized {
     /// Return a mutable reference to the element's property storage.
     fn properties(&mut self) -> &mut Self::Props;
 
+    /// Set the element's text color.
+    fn text_color(mut self, color: Color) -> Self
+    where
+        Self: HasProperty<TextColor>,
+    {
+        *self.property() = Some(TextColor { color });
+        self
+    }
+
+    /// Set the element's text color when disabled.
```

Comment:
It might be nice to link to something user facing for `disabled` here, but iirc there isn't any in current Xilem.